pkg/csi_driver/node.go

```diff
@@ -277,6 +294,9 @@ func (s *nodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolu
 	if mounted {
 		// Already mounted
 		klog.V(4).Infof("NodeStageVolume succeeded on volume %v to staging target path %s, mount already exists.", volumeID, stagingTargetPath)
```

Comment:
this can be a misleading message. we should change it to reflect the right state, i.e path mounted, proceed to configmap updates

Reply:
Done